test/lib/urls.test.ts

```diff
@@ -0,0 +1,49 @@
+import { formatUrlQueryParams } from "../../src/lib/urls";
+
+describe("format URL query params function", () => {
+  let url;
+  let queryParamMap;
```

Comment:
The tests here have just been moved from `utils.test.ts` to `urls.test.ts` as a hangover from a previous move, where I forgot to move the tests as well.

There is one new test highlighted below.